dashboard/app/controllers/levels_controller.rb

```diff
@@ -59,6 +59,8 @@ class LevelsController < ApplicationController
     Weblab
   ]
 
+  use_reader_connection_for_route(:get_rubric)
```

Comment:
nit: I'd prefer for this line to be adjacent to the method definition

Reply:
Oops. Yes. I fixed this mistake in the teacher feedbacks controller, but forgot to apply that to this one.